app/models/jira_project.rb

```diff
@@ -1,30 +0,0 @@
-# == Schema Information
-#
-# Table name: jira_projects
-#
-#  id               :bigint           not null, primary key
-#  deleted_at       :datetime
-#  jira_project_key :string           not null
-#  project_name     :string
-#  created_at       :datetime         not null
-#  updated_at       :datetime         not null
-#  product_id       :bigint
-#
-# Indexes
-#
-#  index_jira_projects_on_product_id  (product_id)
-#
-# Foreign Keys
-#
-#  fk_rails_...  (product_id => products.id)
-#
-
-class JiraProject < ApplicationRecord
```

Comment:
Why are we removing the `JiraProject`? I think it might make sense to keep it (maybe rename it to `JiraBoard`), but  having `JiraIssue` relate to a `Product` doesn't look that good to me. In the future we might even have N JiraBoards for 1 Product...